contents/handbook/company/management.md

```diff
@@ -0,0 +1,57 @@
+
+As we grow, we'll increase the number of managers at PostHog. Here's what a manager at PostHog looks like.
+
+## Defining the role of manager
+
+A manager at PostHog has these tasks
+1. Making sure their direct reports are happy and productive
+1. Setting the right context for direct reports to do their job
+
+That's it.
+
+A manager at PostHog is _not_ responsible for:
+1. Setting compensation (we have transparent compensation)
+1. Setting tasks for their direct reports
+1. Creating a career path (career paths should be transparent and documented, and for now centrally managed)
+1. "Approving," whether that's projects, expenses, days off or accounts (people should have admin access by default to most things)
+
+
+## What does setting context mean?
+
+At PostHog, we exclusively hire people that are the best in their field.
+That means managers won't need to spend time telling their direct reports what to do.
+
+However, for those people to make the best decisions, they need context. That context can be
+- what a customer said was or wasn't important to them
+- what the metrics are saying needs to be improved
+- what another team in the organisation is working on
+- what the overal goals are for PostHog
+
+The shift here, and the biggest difference between PostHog and other places, is that in the end it is up to the individual to make the decisions.
+All you can do as a manager is set context. From there, you'll have to trust that we've made the right hiring decisions and that the individual is able to execute on that. If they can't, we have a [generous severance policy](/handbook/people/compensation#severance).
+
+Decisions aren't just about buying a piece of software or choosing a color for a button. It's also about what to work on, what to invest time in, or where to take entire parts of our product.
+Again, we've hired the best people and have high talent density, so we trust everyone to make these kinds of decisions.
+
+As a manager, it's tempting to see yourself as the sole owner of all the information, and give it out sparingly.
+People will come to you often with questions (because they don't have the context) and when they do you'll get more validation that holding all the context yourself makes you an Important Person.
+What managers should really aim for at PostHog is to make themselves obsolete. Share as much context as possible, preferably in written form. That way everyone will be able to do their best work.
+
+## Part-time managers
+
+Because of the relatively short list of tasks that managers have, management at PostHog is a part-time job.
+That means everyone, from CEO and CTO on up, still spend the majority of their time on practicing what they do best (which likely isn't management!).
+
+As an engineer, you wouldn't respect the opinion of someone who can't code on a coding question.
+As a designer, you really want your manager to have an eye for design.
+That's why it's important for managers to keep practicing their craft.
```

Comment:
It would be great to have an operations example in here - 'As an operator, you want to be managed by someone who has scaled a business'?

I _think_ I'm understanding that the point is not that your manager is better than you at the things you do in a technical sense, but more that they are informed/technically competent enough to set context - is that right?

Reply:
Yep that's right. Again it comes back to the whole 'part-time manager' thing, where I don't believe we should have full time managers for a long time at least. Will add the example